ugs-platform/ugs-platform-plugin-pendant/src/main/java/com/willwinder/ugs/platform/plugin/pendant/Bundle.properties

```diff
@@ -0,0 +1 @@
+OpenIDE-Module-Name=PendantModule
```

Comment:
Please add these fields, they are used in the "Plugins" section to describe the purpose:
```
OpenIDE-Module-Short-Description=
OpenIDE-Module-Long-Description=
OpenIDE-Module-Display-Category=UGS Plugin (or maybe a new category if this is a general purpose interface)
```